package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "go-google-scraper-challenge",
+  "version": "1.0.0",
+  "description": "A project for Nimble Internal Certification on Web: Go",
```

Should we update this to be consistent with the README?
```suggestion
  "description": "A project for Nimble Go Internal Certification on Web",
```